setupext.py

```diff
@@ -1493,6 +1493,8 @@ def add_flags(self, ext):
         if sys.platform == 'win32':
             # PSAPI library needed for finding Tcl / Tk at run time
             ext.libraries.extend(['psapi'])
+        elif sys.platform != 'darwin':
```

Comment:
seems to be a slightly obfuscated way to say "startswith('linux')"...

Reply:
I had assumed it was needed on BSD, but I think that was wrong now.

Reply:
Fixed.